lib/dns/proof.js

```diff
@@ -0,0 +1,9 @@
+class Proof{
```

Comment:
What is this here for?

Reply:
This is used at `buildProofs` function to construct an object which contains data to submit into DNSSEC contract